test/index_tests.js

```diff
@@ -310,7 +310,9 @@ it('WHEN mongoose models are supplied THEN the definitions and tags should be in
     'api-spec.json',
     1000,
     'custom-docs',
-    mongooseModels
+    mongooseModels,
+    mongooseModels,
```

Comment:
duplicate `mongooseModels`?

Reply:
Nope, that parameter is for `tags` 😄. Need to suply it to introduce `ignoreNodeEnvironments`. Removing them anyway, as they are the default values.